apps/gallery/test/marionette/info_view_test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var Gallery = require('./lib/gallery.js'),
+    Fullscreen_View = require('./lib/fullscreen_view.js'),
+    GalleryInfoView = require('./lib/gallery_info_view.js'),
+    assert = require('assert');
+
+marionette('the gallery', function() {
```

Comment:
nit: 'the gallery' can be changed to more informative text such as 'Gallery Info View tests'